### ISSUE TITLE
Using startup_script with entry_point

### ISSUE THREAD

**Issue by navneethc** [python-cmd2/cmd2#1029]
I'm sorry if this kind of question doesn't belond in the 'Issues' category. I've tried looking through the documentation and searching Google, but I couldn't find an answer. I'm using this as a last resort.
<hr>

Could someone please show me a way to invoke the `startup_script` when one wants to run the script from an entry point?

This is how my source directory looks:

```.
├── config
├── script.py
├── __init__.py
└── utils
```

The `startup_script` is inside `config`. `script.py` both invokes the `startup_script` and also contains the entry point. When a wheel is created and the package is installed, a copy of `script.py` is placed under `bin` in my environment, but this file doesn't have access to the startup script which is under `site-packages`. 

Basically, I would like to run the CLI as `script`, which is the name for the entry point, from anywhere in the environment after installing it, rather than `python script.py` from the source directory, in such a way that the CLI can read the startup script.

**Comment by navneethc** [comment]
Closing this issue as a solution was found, and the original problem was found to be not related to cmd2.
